Owl/Models/Chat/NewMessage.swift

```diff
@@ -9,5 +9,5 @@ import Foundation
 
 struct NewMessage {
     let chatId: String
-    let message: Message
+    let message: MessageRequest
```

Comment:
I think u don't need `NewMessage` struct. U can add chatId property to `MessageRequest` struct